src/i18n/i18n.ts

```diff
@@ -128,13 +128,13 @@ const titleCasePostProcessor: PostProcessorModule = {
 const sentenceCasePostProcessor: PostProcessorModule = {
     type: 'postProcessor',
     name: 'sentenceCase',
-    process: (value: string) => {
+    process: (value: string, _: string, __: TOptions<StringMap>, translator: any) => {
         const sentences = value.split('. ');
 
         return sentences
             .map((sentence) => {
                 return (
-                    sentence.charAt(0).toLocaleUpperCase() + sentence.slice(1).toLocaleLowerCase()
+                    sentence.charAt(0).toLocaleUpperCase() + (translator.language !== 'de' ? sentence.slice(1).toLocaleLowerCase() : sentence.slice(1))
```

Comment:
Since you don't want to perform any postprocessing on the string at all for `de`, you should move the conditional logic to the start of the function for the sake of readability and performance. Having the ternary operator inside the loop would affect performance (albeit very little) on `de` and other languages when it doesn't need to.

Like:
```typescript
    process: (value, _key, _options, translator) => {
        if (translator.language === 'de') {
            return sentence;
        }

        # rest of the function
    },
```

Reply:
I did initially write it like that, but when testing this caused the english-language strings (strings without existing german translation) to be lowercase as sentenceCase didn't make their first character uppercase and `translator.language` was `de` no matter what.

 That is why I decided on this approach. Should I still change it?

Reply:
No you should keep the logic as is then. I forgot about the fallback language case.

I do want to make the logic more re-usable though.

Something like
```
const ignoreSentenceCaseLanguages = ['de'];

# And then update the ternary condition
!ignoreSentenceCaseLanguages.includes(translator.language) ? ... : ....
```